From 6e7281ded075bf89f1ca324da35790f1336edaec Mon Sep 17 00:00:00 2001
From: Philip Balister <philip@balister.org>
Date: Tue, 2 Nov 2021 15:28:33 -0400
Subject: [PATCH] From bc9476ad859b7e582b89f532f5e84771d6e026d2 Mon Sep 17
 00:00:00 2001 From: Yongjoo Ahn <yongjoo1.ahn@samsung.com> Date: Fri, 16 Oct
 2020 14:22:38 +0900 Subject: [PATCH] Modify makefile for yocto

- Modify makefile for yocto make build support

Signed-off-by: Yongjoo Ahn <yongjoo1.ahn@samsung.com>

Updated by:

Signed-off-by: Philip Balister <philip@balister.org>
---
 Makefile.install | 4 ++--
 Makefile.system  | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/Makefile.install b/Makefile.install
index 28727de3..605125fb 100644
--- a/Makefile.install
+++ b/Makefile.install
@@ -151,8 +151,8 @@ endif
 
 #Generating openblas.pc
 	@echo Generating $(LIBSONAMEBASE).pc in "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)"
-	@echo 'libdir='$(OPENBLAS_LIBRARY_DIR) > "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
-	@echo 'includedir='$(OPENBLAS_INCLUDE_DIR) >> "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
+	@echo 'libdir='$(libdir) > "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
+	@echo 'includedir='$(includedir) >> "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
 	@echo 'openblas_config= USE_64BITINT='$(USE_64BITINT) 'DYNAMIC_ARCH='$(DYNAMIC_ARCH) 'DYNAMIC_OLDER='$(DYNAMIC_OLDER) 'NO_CBLAS='$(NO_CBLAS) 'NO_LAPACK='$(NO_LAPACK) 'NO_LAPACKE='$(NO_LAPACKE) 'NO_AFFINITY='$(NO_AFFINITY) 'USE_OPENMP='$(USE_OPENMP) $(CORE) 'MAX_THREADS='$(NUM_THREADS)>> "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
 	@echo 'version='$(VERSION) >> "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
 	@echo 'extralib='$(PKG_EXTRALIB) >> "$(DESTDIR)$(OPENBLAS_PKGCONFIG_DIR)/$(LIBSONAMEBASE).pc"
diff --git a/Makefile.system b/Makefile.system
index 833511fa..b9958816 100644
--- a/Makefile.system
+++ b/Makefile.system
@@ -1402,7 +1402,7 @@ endif
 
 KERNELDIR	= $(TOPDIR)/kernel/$(ARCH)
 
-include $(TOPDIR)/Makefile.$(ARCH)
+#include $(TOPDIR)/Makefile.$(ARCH)
 
 ifneq ($(C_COMPILER), PGI)
 ifneq ($(C_COMPILER), SUN)
-- 
2.31.1

